components/Map.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { MapView, Location, Permissions, Marker, TouchableHighlight, View, Text } from 'expo';
+
+import * as api from "../api";
+
+export default class Map extends React.Component {
+    state = {
+        mapRegion: { latitude: 37.78825, longitude: -122.4324, latitudeDelta: 0.0922, longitudeDelta: 0.0421 },
+        locationResult: null,
+        location: {coords: { latitude: 37.78825, longitude: -122.4324}},
+        markers: [],
+        pins : [
+          ]
+      };
+  render() {
+    return (
+      <MapView 
+      style={{ alignSelf: 'stretch', height: 550 }}
+      region={{ latitude: this.state.location.coords.latitude, longitude: this.state.location.coords.longitude, latitudeDelta: 0.0922, longitudeDelta: 0.0421 }}
+      showsUserLocation={true}
+      // onRegionChangeComplete={this._handleMapRegionChange}
+      // onPanDrag={this._handleDrag}
+      >
+      {this.state.pins.map(pin => (
+        <MapView.Marker
+        key={pin.pin_id}
+coordinate={{latitude: Number(pin.latitude),
+    longitude: Number(pin.longitude)}}
+    title={pin.creator}
+    description={pin.timestamp}
+    onPress={() => this.props.navigation.navigate("Pin", {
+      pin: pin
+    })}
+>
+</MapView.Marker>
+      ))}
+
+      </MapView>
+    );
+  }
+
+  componentDidMount() {
+    this._getLocationAsync();
+    this.fetchPins();
+  }
+
+//   _handleMapRegionChange = mapRegion => {
+//     this.setState({ mapRegion });
+//   };
+
+//   _handleDrag = () => {
+// this.setState({location: mapRegion })
+//   }
```

Comment:
Do we need this code?

Reply:
Not the commented out code, apologies will delete and make a new commit